concepts/adaptation/spans_september_2023.csv

```diff
@@ -0,0 +1,2026 @@
+document_id,text_block_text_hash,type,id,text,start_idx,end_idx,sentence,pred_probability,annotator
+UNFCCC.non-party.1750.0,87f2a4feef72a378f1face8144390f49__f198bc193a37008f2fea65d0d95adbc16288127074a4e262a03aa9a805aee534,ADAPTATION,ADAPTATION,adaptation,555,565,"These arrangements are designed at the national level and are part of the government structure that governs and monitors the progress of the activities and projects that will be implemented to achieve the mitigation and adaptation targets in the enhanced NDC, as outlined in the Enhanced NDC Implementation Plan (2021–2030).",1.0,explorer
```

Comment:
Again out of interest is this data basically all the passages that we deem to reference `adaptation` in the dataset?

Reply:
yeah - just the new 14 docs. all the spans csv files in the folder are all the passages in the entire dataset